packages/usa-in-page-navigation/src/test/template.html

```diff
@@ -0,0 +1,23 @@
+<div class="usa-in-page-nav-container">
+  <main id="main-content" class="main-content">
+    <h1>In page navigation test heading</h1>
+
+    <h2><a id="section_0" class="usa-anchor-tag"></a>Section 1</h2>
+    <p></p>
+    <h3><a id="section_1" class="usa-anchor-tag"></a>Section 1.1</h3>
+    <p>Section 1.1 content</p>
+    <h3><a id="section_2" class="usa-anchor-tag"></a>Section 1.2</h3>
+    <p>Section 1.2. content</p>
+  </main>
+
+  <aside class="usa-in-page-nav">
+    <nav aria-label="In-page navigation">
+      <h4 class="site-subheading">On this page</h4>
+      <ul class="usa-in-page-nav-list">
+        <li class="usa-in-page-nav__item"><a href="#section_0" class="usa-in-page-nav__link usa-current">Section 1</a></li>
+        <li class="usa-in-page-nav__item sub-item"><a href="#section_1" class="usa-in-page-nav__link">Section 1.1</a></li>
+        <li class="usa-in-page-nav__item sub-item"><a href="#section_2" class="usa-in-page-nav__link">Section 1.2</a></li>
```

Comment:
Looks like these items missed the BEM class update described [here](https://github.com/uswds/uswds/pull/4918#issuecomment-1301502261).

> * `usa-in-page-nav-list` → `usa-in-page-nav__list`
> * `site-subheading` → `usa-in-page-nav__heading`
> * `sub-item` → `usa-in-page-nav__item--sub-item`

Reply:
Ah... forgot to update the tests!